core/src/main/java/io/questdb/std/Files.java

```diff
@@ -48,6 +48,7 @@ public final class Files {
     public static final int FILES_RENAME_OK = 0;
     public static final int MAP_RO = 1;
     public static final int MAP_RW = 2;
+    public static final int NFS_MAGIC = 0x6969;
```

Comment:
Where did you get the magic value for NFS? I suspect it may differ from OS to OS

Reply:
https://github.com/torvalds/linux/blob/e9d22f7a6655941fc8b2b942ed354ec780936b3e/include/uapi/linux/magic.h#L51

I think Linux is where it matters the most (in practice). Are you worried about false positives on other OSs? I can I could guard it with `Os.isLinux()` or something.

Reply:
Can you that it works on Linux inside docker with a mapped directory? This is quite mainstream.